android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -400,6 +424,8 @@ public void peerConnectionInit(ReadableMap configuration, int id) {
         PeerConnection.RTCConfiguration rtcConfiguration = parseRTCConfiguration(configuration);
 
         try {
+            ThreadUtils.runOnExecutor(() -> peerConnectionInitAsync(rtcConfiguration, id));
```

Comment:
This seems to duplicate the same code as below, but isn't run synchronously. Seems unneeded.